server/channels/app/support_packet.go

```diff
@@ -123,8 +123,16 @@ func (a *App) generateSupportPacketYaml(c request.CTX) (*model.FileData, error)
 	/* LDAP */
 
 	var vendorName, vendorVersion string
-	if ldapInterface := a.Ldap(); ldapInterface != nil {
-		vendorName, vendorVersion = ldapInterface.GetVendorNameAndVendorVersion(c)
+	ldap := a.Ldap()
+	if ldap != nil {
+		vendorName, vendorVersion, err = ldap.GetVendorNameAndVendorVersion(c)
+		if err != nil {
+			rErr = multierror.Append(errors.Wrap(err, "error while getting LDAP vendor info"))
+		}
+		if vendorName == "" && vendorVersion == "" {
+			vendorName = "unknown"
```

Comment:
nit: extract unknown as a constant to avoid magic strings

Reply:
Where would you place the constant? In the local or global namespace?